addons/website_slides/views/slide_channel_partner_views.xml

```diff
@@ -32,5 +25,13 @@
                 </tree>
             </field>
         </record>
+
+        <record id="slide_channel_partner_action" model="ir.actions.act_window">
```

Comment:
Why this change ? Normally defining manually the tree view_id is not necessary as we have only one default one.

Reply:
Indeed, I implemented another view for slide.channel.partner and forgot to remove that line after changing it :)

Reply:
done !